Don't use the CPP define for fileno in stdio.h
It causes clang++ to blow up.

Index: src/utils/file.cpp
--- src/utils/file.cpp.orig
+++ src/utils/file.cpp
@@ -46,6 +46,7 @@ file_ptr::operator int() {
   return static_cast<const file_ptr&>(*this);
 }
 file_ptr::operator int() const {
+#undef fileno
   return fileno(*this);
 }
 
